README.md

```diff
@@ -1,120 +1,65 @@
-Node Tachyon
-============
+<table width="100%">
+	<tr>
+		<td align="left" colspan="2">
+			<strong>Tachyon</strong><br />
+			Faster than light image resizing service that runs on AWS. Super simple to set up, highly available and very performant.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			A <strong><a href="https://hmn.md/">Human Made</a></strong> project. Maintained by @joehoyle.
+		</td>
+		<td align="center">
+			<img src="https://hmn.md/content/themes/hmnmd/assets/images/hm-logo.svg" width="100" />
+		</td>
+	</tr>
+</table>
 
-A fast image processor to be used with the Tachyon [WordPress plugin](https://github.com/humanmade/tachyon-plugin). node-tachyon can either be run directly in NodeJS on your server, or also utilize AWS Lambda and act as a proxy to the Lambda function.
 
+Tachyon is built with some strong opinions and assumptions:
 
-## Installation
+- Runs on AWS (using CloudFront, Lambda and API Gateway.)
+- Expects original image files to be stored on Amazon S3.
+- Only supports simple image resizing, not a full image manipulation service.
 
-If you're running on OSX make sure you have libvips installed, the easiest way is to use homebrew:
+Tachyon works best with WordPress, coupled with [S3 Uploads](github.com/humanmade/s3-uploads) and the [Tachyon Plugin](https://github.com/humanmade/tachyon-plugin).
 
-```
-brew install homebrew/science/vips --with-webp --with-graphicsmagick
-```
+![](http://joehoyle-captured.s3.amazonaws.com/I77brfBv.png)
```

Comment:
Before I merge this, will add this to the Engineering Handbook so it has a more permanent URL